### ISSUE TITLE
Uncaught ReferenceError: process is not defined

### ISSUE THREAD

**Issue by MattiaPontonioKineton** [rdkcentral/Lightning-CLI#24]
Hi,
I'm trying to use https://xstate.js.org/docs/ and I'm experiencing an issue:
```
environment.js:3 Uncaught ReferenceError: process is not defined
    at environment.js:3
    at createCommonjsModule (index.js:9)
    at constants.js:6
    at index.js:6
(anonymous) @ environment.js:3
createCommonjsModule @ index.js:9
(anonymous) @ constants.js:6
(anonymous) @ index.js:6
```
Seems this is connected with Rollup.
Here some similar issues:

- https://github.com/davidkpiano/xstate/issues/706
- https://github.com/davidkpiano/xstate/issues/443

## The file
- https://github.com/davidkpiano/xstate/blob/master/packages/core/src/environment.ts

Can You fix this in the CLI or Can I extends the configuration of Rollup?

**Comment by MattiaPontonioKineton** [comment]
This should fix the problem, I've tested in the global package (install the dependency):
```
const babel = require('rollup-plugin-babel')
const resolve = require('rollup-plugin-node-resolve')
const commonjs = require('rollup-plugin-commonjs')
const babelPresentEnv = require('@babel/preset-env')
const babelPluginTransFormSpread = require('@babel/plugin-transform-spread')
const babelPluginTransFormParameters = require('@babel/plugin-transform-parameters')
import replace from '@rollup/plugin-replace'

module.exports = {
  plugins: [
    resolve({
      mainFields: ['main', 'browser']
    }),
    commonjs(),
    babel({
      presets: [
        [
          babelPresentEnv,
          {
            targets: {
              chrome: '44',
            },
            spec: true,
            debug: false,
            useBuiltIns: 'usage',
            corejs: '^2.6.11',
          },
        ],
      ],
      plugins: [babelPluginTransFormSpread, babelPluginTransFormParameters],
    }),
    replace({
      'process.env.NODE_ENV': JSON.stringify('production')
    })
  ],
  output: {
    format: 'iife',
    sourcemap: true,
  },
}

```
Can I submit the code?
```
$ git push origin 24
remote: Permission to WebPlatformForEmbedded/Lightning-CLI.git denied to MattiaPontonioKineton.
fatal: unable to access 'https://github.com/WebPlatformForEmbedded/Lightning-CLI.git/': The requested URL returned error: 403
```

**Comment by MattiaPontonioKineton** [comment]
Hello @michielvandergeest,
any updates on this issue?
Updating the CLI breaking my developer experience today in my project with `wpe-lightning` + `xstate`.
I think that should break even other use of other libraries in the wpe-lightning modules.
Best regards :)
Mattia

**Comment by michielvandergeest** [comment]
This should be fixed in release 1.7.0
